### ISSUE TITLE
Move ne pop_max filter logic into min zoom

### ISSUE THREAD

**Issue by rmarianski** [tilezen/vector-datasource#1369]
In the ne populated places query, there are [explicit checks for the population](https://github.com/tilezen/vector-datasource/blob/4d4628244181efe0a8fefebc3daf9d90f0adc942/queries/ne-places.jinja2#L33-L39) in the where clause. Can these be moved into the min zoom calculation instead?

**Comment by nvkelso** [comment]
Probably? They're there to ensure smaller regionally important cities from Natural Earth still show up when we switch over to OpenStreetMap for the first few zooms (which are just "big" cities).

Associated "remainder" tests are here:

- https://github.com/tilezen/vector-datasource/blob/master/integration-test/981-remove-unstyled-ne-places.py#L41-L57

**Comment by rmarianski** [comment]
Ok, I'm just coming from the perspective of trying to normalize the queries further. I'll give it a shot.

**Comment by rmarianski** [comment]
I started down this path, and began looking at updating the yaml for the min zoom calculation. It's a [lookup table](https://github.com/tilezen/vector-datasource/blob/5bb23c587fc5a959c240065268385f0e53b5e34c/yaml/places.yaml#L27-L39), and unless I"m missing a way to combine some of the operators we have, we'll need to make a new function and implement it in sql as well to contain this logic. @zerebubuth that sound right to you?

**Comment by zerebubuth** [comment]
Yup. Using `lookup` sounds good to me. I'm not sure what extra logic is needed to combine this with `min_zoom` - is there something in addition to taking the `max` of the `lookup` and whatever the previous `min_zoom` calculation was?

**Comment by rmarianski** [comment]
I was hoping for a quick win here, but it looks like to actually move this to yaml would require us to grow a max zoom concept. In particular, we need to have the ability for some features to selectively disappear at higher zoom levels based on population.
We could just go with min zoom and filter out some of the data in a downstream transform, but I don't think that's the direction we want to go in.

**Comment by nvkelso** [comment]
Suggest tabling this for now.
